src/resources/platform-adaptations.md

```diff
@@ -296,6 +296,11 @@ Therefore a fallback font is used when running on Android
 if the platform is debug-overridden to iOS or the
```

Comment:
I am not sure... 

@MitchellGoodwin could you explain by chance?

Reply:
I think these means if the platform is set by the code rather than by letting Flutter determine which platform it's running in. So if you add to your theme `platform: TargetPlatform.iOS`